src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -527,4 +527,10 @@
   <data name="Convert_to_for" xml:space="preserve">
     <value>Convert to 'for'</value>
   </data>
+  <data name="Convert_to_regular_method" xml:space="preserve">
+    <value>Convert to regular method</value>
+  </data>
+  <data name="Convert_to_method" xml:space="preserve">
+    <value>Convert to method</value>
```

Comment:
what is the difference between a method and a regular method?

Reply:
Wait, I recall reverting to "Convert to method". The previous string is not supposed to be there.